codalab/rest/users.py

```diff
@@ -63,7 +67,10 @@ def fetch_authenticated_user():
 def update_authenticated_user():
     """Update one or multiple fields of the authenticated user."""
     # Load update request data
-    user_info, errors = AuthenticatedUserSchema(strict=True).load(request.json, partial=True)
+    user_info = AuthenticatedUserSchema(
+        strict=True,
+        dump_only=USER_READ_ONLY_FIELDS,
```

Comment:
Wait, so dump_only refers to fields that *won't* be dumped (serialized)?  That logically makes sense, but the naming is a bit confusing.